src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/zipfilestatus/ZipFileStatusService.java

```diff
@@ -79,6 +80,24 @@ public ZipFileStatus getStatusFor(String zipFileName) {
         return zipFileStatusList;
     }
 
+    public ZipFileStatus getStatusByCcdId(String ccdId) {
```

Comment:
It should return `CcdIdStatus` extending `ZipFileStatus` with added `String ccdId` attribute. In this case the resulting JSon will be displaying the search criteria along with the result; this will be in line with searching by `fileName` - the search criteria is provided disregarding do we know the file or not.

Reply:
I think we do not need to extend.